bonjour.js

```diff
@@ -77,6 +57,20 @@ app.use(function(req, res, next) {
     next();
 });
 
+app.use((req, res, next) => {
+    const serverSpan = opentracing.globalTracer().startSpan("GET", {
+        childOf: extractSpanContext(opentracing.globalTracer(), req.headers),
+        tags: {
+            'http.method': req.method,
+            'http.url': extractUrl(req),
```

Comment:
`opentracing.Tags.HTTP_URL` should work (but please verify I'm not sure how it is exported)